modules/optimization/test/tests/optimizationreporter/material/adjoint_iteration_output.i

```diff
@@ -0,0 +1,112 @@
+[Mesh]
```

Comment:
keep the test here for now until the design is set

but if we stay with this "ExodusOptimizationSteady" object they will need to be moved to a 
optizimation/test/tests/outputs/exodus_optimization_steady folder.